tools/src/test/java/com/nedap/archie/json/flat/FlatJsonGeneratorTest.java

```diff
@@ -0,0 +1,123 @@
+package com.nedap.archie.json.flat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.nedap.archie.adlparser.ADLParser;
+import com.nedap.archie.aom.Archetype;
+import com.nedap.archie.aom.OperationalTemplate;
+import com.nedap.archie.creation.ExampleJsonInstanceGenerator;
+import com.nedap.archie.flattener.Flattener;
+import com.nedap.archie.flattener.FlattenerConfiguration;
+import com.nedap.archie.flattener.SimpleArchetypeRepository;
+import com.nedap.archie.json.JacksonUtil;
+import com.nedap.archie.rm.RMObject;
+import com.nedap.archie.rminfo.ArchieRMInfoLookup;
+import org.junit.Test;
+import org.openehr.referencemodels.BuiltinReferenceModels;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class FlatJsonGeneratorTest {
+
+    private static final String BLOOD_PRESSURE_PATH = "/ckm-mirror/local/archetypes/entry/observation/openEHR-EHR-OBSERVATION.blood_pressure.v1.1.0.adls";
+    private static final double EPSILON = 0.00000001d;
```

Comment:
Why call this variable EPSILON? The function itself calls it delta right?

Reply:
Because the variable defining the maximum difference between two floating point number to be still considered to be equal is usually called epsilon. You could also call it delta, as in difference. Both are used.